code/modules/multiz/turf.dm

```diff
@@ -52,6 +52,7 @@
 	below = GetBelow(src)
 	turf_changed_event.register(below, src, /turf/simulated/open/update_icon)
 	levelupdate()
+	below.update_icon() // So the 'ceiling-less' overlay gets added.
```

Comment:
Perhaps instead of adding this  line, you could change line 58 below to be `OS_controller.add_turf(below, TRUE)`  that way the update_icons will be managed by the openspace controller.  This could be especially useful in the case of a 3+ story high openspace area.